target/linux/qualcommax/config-6.1

```diff
@@ -28,6 +28,7 @@ CONFIG_ARM64_ERRATUM_2054223=y
 CONFIG_ARM64_ERRATUM_2067961=y
 CONFIG_ARM64_ERRATUM_2077057=y
 CONFIG_ARM64_ERRATUM_2658417=y
+CONFIG_ARM64_ERRATUM_2966298=y
```

Comment:
Qualcommax is using Cortex-A53 cores, so it shouldn't be affected by this at all

Reply:
Same goes for all targets in OpenWrt, none of them uses A520 core, it probably makes sense to enable it for armsr only though that is targeting ARMv8

Reply:
At least two checks failed which prompted me to look into the config changes.  I cannot remember which two they were though.  I think one as `armsr/armv8`... I will remove the modifications and see which ones fail.

Reply:
They are just using the default value, for OpenWrt it should just be disabled in the generic config.

Reply:
(aka setting this as # ... is not defined in the generic configuration)

Reply:
Yes, and then you can drop all of the target config changes as we dont have that core in any OpenWrt target.

Reply:
Done.